qemu/tests/qemu_guest_agent.py

```diff
@@ -1026,6 +1027,100 @@ def gagent_check_with_fsfreeze(self, test, params, env):
         cmd_del_file = "%s %s" % (params["cmd_del"], tmp_file)
         session.cmd(cmd_del_file)
 
+    @error_context.context_aware
+    def gagent_check_guest_exec(self, test, params, env):
+        """
+        Execute a command in the guest via guest-exec cmd,
+        and check status of this process.
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        """
+
```

Comment:
Please add steps for this case

Reply:
Thanks, I add the steps.
Pls have a check.
Host_RHEL.mx.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.x.0.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_guest_exec: PASS (51.21 s)